### DISCUSSION TITLE
Expanded service scope

### DISCUSSION THREAD

**Discussion by kecrily** [FastGitORG/discussion#29] [Q&A] [answered]
Does FastGit UK have plans to offer services for other regions that are having trouble accessing GitHub?

In addition to China mainland, I understand there are difficulties accessing GitHub in Iran, Myanmar, North Korea, etc. I think it would be great to support them

**Comment by KevinZonda** [comment, marked as answer]
They can access, unlimit them.

**Reply by KevinZonda** [reply]
(if they hv internet
